front/src/trackview/Trackview.js

```diff
@@ -156,7 +156,7 @@ const Trackview = (props) => {
         </Grid>
       )}
       {visible.infobox && ( // extra info of the track
-        <div>
+        <div style={{whiteSpace: 'pre-wrap'}}>

```

Comment:
It would be better to not use inline styling. Add a fitting className, and then add css rule in Trackview.module.scss.